lib/localeapp/cli/install.rb

```diff
@@ -2,35 +2,39 @@ module Localeapp
   module CLI
     class Install < Command
       attr_accessor :config_type
+      attr_accessor :write_dotenv
```

Comment:
The attribute has been removed when introducing the command line option and abandoning `.env` detection.